src/components/Dashboard/TodoList.vue

```diff
@@ -668,7 +668,7 @@ export default class TodoList extends Mixins(
 
   /** show action required only for SP/GP with compaince warning. */
```

Comment:
Typo.
Also capitalize. (This is a JSDoc comment that is displayed when you hover on the getter.)